ime/app/src/test/java/com/anysoftkeyboard/ime/AnySoftKeyboardGestureTypingTest.java

```diff
@@ -138,6 +138,20 @@ public void testOutputPrimarySuggestionOnGestureDone() {
         Assert.assertEquals("hello", mAnySoftKeyboardUnderTest.getCurrentInputConnectionText());
     }
 
+    @Test
+    public void testOutputCapitalisedOnShiftLocked() {
```

Comment:
Nice! For completion, can you do a test for not-shifted (although it is covered by other unit tests)?

Reply:
This would be identical to the test above (`testOutputPrimarySuggestionOnGestureDone`). I can rename that test if you like?

Reply:
no, this is good. I missed that.